### PR TITLE
5060 Overwiew Application Extension MapCatalog

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description
Improvement of the User Guide with the addition, in Managing Context section, of the following sub-sections :

-  Overview
- Application context 
- Extensions Library
- Map Catalog

## Other useful information
Related to #5060 
[ReadTheDoc](https://elena-mapstore-docs.readthedocs.io/en/5060_four_sections/user-guide/managing-contexts/)

### CODE REVIEW

File: docs/user-guide/map-catalog.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    The added map replaces the context, in this way the context extensions are no more available.
+
+For each map in the *Map Catalog* list the following buttons are displayed:
+
+<img src="../img/map-catalog/buttons-on-map-catalog.jpg" class="ms-docimage"/>
+
+* The **Delete** button <img src="../img/button/delete_button.jpg" class="ms-docbutton" /> allows the user to remove the map
+
+!!! warning
+    The map deleted from the *Map Catalog* remains available on the MapStore [Homepage](https://mapstore.geo-solutions.it/mapstore/#/).
```
Comment (tdipisa) [inline docs/user-guide/map-catalog.md:22]: **Remove this warning**, the map list is the same betweek the homepage and the MapCatalog

>If you found something like this, generally speaking this is a bug and not something to document! It is working well anyway, just tested, so remove the warning.

File: docs/user-guide/map-catalog.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+* The **Edit Properties** button <img src="../img/button/edit_button.jpg" class="ms-docbutton" /> allows the user to [Edit Properties](resources-properties.md) of the map
+
+* the **Share** button <img src="../img/button/share.jpg" class="ms-docbutton" /> allows the user to [share](share.md) the map
```
Comment (tdipisa) [inline docs/user-guide/map-catalog.md:26]: The **Share** ...

File: docs/user-guide/map-catalog.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+!!! note
```
Comment (tdipisa) [inline docs/user-guide/map-catalog.md:28]: This note can be removed I think...

**Comment by ElenaGallo** [comment]
> @ElenaGallo there are a lot of deleted files, why? 

@tdipisa They had already been deleted when I updated the branch, so I only committed my change to the PR

**Comment by tdipisa** [comment]
> > @ElenaGallo there are a lot of deleted files, why?
> 
> @tdipisa They had already been deleted when I updated the branch, so I only committed my change to the PR

@ElenaGallo the files are needed for the release process, therefore I think you should restore them. @offtherailz can you please confirm that these files are still needed?

**Pull request closed without merging**